From: =?utf-8?q?Jeremy_B=C3=ADcha?= <jeremy.bicha@canonical.com>
Date: Fri, 6 Dec 2024 13:53:00 -0500
Subject: Revert "build: Add wrap file for GStreamer"

This reverts commit 60d46ccc6c25af13787cb55a72702f9dc27d6093.

It duplicates our gstreamer-player subproject
---
 subprojects/gstreamer-full.wrap | 8 --------
 1 file changed, 8 deletions(-)
 delete mode 100644 subprojects/gstreamer-full.wrap

diff --git a/subprojects/gstreamer-full.wrap b/subprojects/gstreamer-full.wrap
deleted file mode 100644
index 1ca3757..0000000
--- a/subprojects/gstreamer-full.wrap
+++ /dev/null
@@ -1,8 +0,0 @@
-[wrap-git]
-directory=gstreamer
-url=https://gitlab.freedesktop.org/gstreamer/gstreamer.git
-revision=1.24
-depth=1
-
-[provide]
-dependency_names = gstreamer-1.0, gstreamer-player-1.0, gstreamer-gl-1.0
\ No newline at end of file
